docs/Authentication.md

```diff
@@ -73,7 +73,7 @@ For instance, to pass the token obtained during login as an `Authorization` head
 
 ```js
 import { simpleRestClient, fetchUtils, Admin, Resource } from 'admin-on-rest';
-const httpClient = (url, options) => {
+const httpClient = (url, options={}) => {
```

Comment:
haha for such a small commit I didn't get in the trouble of installing eslint... and shamefully failed !